pkgs/os-specific/bsd/netbsd/default.nix

```diff
@@ -414,7 +453,16 @@ let
     path = "include";
     version = "8.0";
     sha256 = "128m77k16i7frvk8kifhmxzk7a37m7z1s0bbmja3ywga6sx6v6sq";
-    nativeBuildInputs = [ makeMinimal install mandoc groff nbperf rpcgen ];
+    nativeBuildInputs = [
+      bsdSetupHook netbsdSetupHook
+      makeMinimal
+      install mandoc groff nbperf rpcgen
+    ];
+    # Hack around coreutils from wrapper clobbering some things
```

Comment:
Does this mean from stdenv.cc?

Reply:
Ah oops I can get rid of that. I needed it before because I had `depsBuildBuild = [ buildPackages.stdenv.cc ]`, which unfortunately changes the PATH in its setup hook. But I just rid of that, because I need `RPCGEN_CPP` anyways to fix that rpcgen issue.